user-guide/Advanced_Modules/Automation_module/FAQ/How_do_I_abort_a_running_Automation_script.md

```diff
@@ -4,12 +4,27 @@ uid: How_do_I_abort_a_running_Automation_script
 
 # How do I abort a running Automation script?
 
+## [From DataMiner 10.2.12/10.3.0 onwards](#tab/tabid-1)
+
+When a dialog box has the focus, you can abort a running Automation script in either of the following ways:
+
+- Close the dialog box by clicking the *X* in the top-right corner.
+
+- Press ALT+F4.
```

Comment:
I would merge the Important remark and this option as one.
Something like: "use the keyboard shortcut. In client app, you can use Alt+F4. In web app, you can use Esc key.